pkg/stanza/operator/helper/multiline.go

```diff
@@ -349,35 +349,44 @@ func trimWhitespaces(data []byte) []byte {
 
 // SplitterConfig consolidates MultilineConfig and FlusherConfig
 type SplitterConfig struct {
+	Encoding  EncodingConfig  `mapstructure:",squash,omitempty"                        json:",inline,omitempty"                       yaml:",inline,omitempty"`
```

Comment:
maybe this should be called `EncodingCfg` or something like it to differentiate from line 389